src/exchange/providers/ollama.py

```diff
@@ -32,15 +32,14 @@ def __init__(self, client: httpx.Client) -> None:
 
     @classmethod
     def from_env(cls: Type["OllamaProvider"]) -> "OllamaProvider":
-        url = os.environ.get("OLLAMA_HOST", OLLAMA_HOST)
-        client = httpx.Client(
-            base_url=url,
-            timeout=httpx.Timeout(60 * 10),
-        )
+        ollama_url = os.environ.get("OLLAMA_HOST", OLLAMA_HOST)
```

Comment:
@baxen lemme know if you think this is more sensible than before

Reply:
yeah this looks good!